doc/source/qibo.rst

```diff
@@ -241,23 +501,62 @@ Trotter decomposition. The Hamiltonians represented by this object are sums of
 commuting terms, following the description of Sec. 4.1 of
 `arXiv:1901.05824 <https://arxiv.org/abs/1901.05824>`_.
 
+Trotter hamiltonian
```

Comment:
```suggestion
Trotter Hamiltonian
```
I would also move the text right above (including the reference) under the "Trotter Hamiltonian" section as it refers to Trotter Hamiltonians.